datasets/il/mod_crypto/crawler.py

```diff
@@ -1,155 +1,154 @@
+        h.apply_date(sanction, "startDate", row.pop("start_date"))
+        # "Last Updated" column in the table of releases
+        h.apply_date(sanction, "modifiedAt", row.pop("last_updated"))
+        # "Validity of Issue" column in the table of releases
+        h.apply_date(sanction, "endDate", row.pop("end_date"))
         if h.is_active(sanction):
             wallet.add("topics", "crime.terror")
-        sanction.add("sourceUrl", source_url)
+        # "File Type" column in the table of releases
+        # e.g., "​Seizure order (ASO 16/25) of the Minister of Defense"
+        sanction.add("sourceUrl", row.pop("order_url"))
+        # Links from the "Validity of Issue" column in the table of releases
+        # e.g., "​Forfeiture Order (FO​ 18/24)"
+        sanction.add("sourceUrl", row.pop("forfeiture_order_url"))
+        # Links from the "File Type" column in the table of releases
+        # e.g., "Annex of the Seizure Order (ASO - 56/23) - Wallet Details"
+        sanction.add("sourceUrl", row.pop("annex_url"))
         context.emit(wallet)
         context.emit(sanction)
 
-    context.audit_data(
-        row,
-        ignore=[
-            # Just sequencing in the spreadsheet, not from the source.
-            "#",
-            # The wording is "the property of the organization, or used in financing
-            # terrorism", i.e. something in the doc belongs to the org, but not
-            # necessarily everything.
-            "Organization",
-        ],
-    )
+    context.audit_data(row)
 
 
 def crawl(context: Context):
     # Get a warning when a notice has been issued
     content_xpath = ".//main"
     doc = fetch_html(context, context.dataset.url, content_xpath, cache_days=1)
     container = doc.xpath(content_xpath)[0]
-
+    # Write a CSV snapshot to check the diff manually (git diff).
+    # Review for any new releases or persons/wallets added (top or bottom of table).
     # The key things to check are
     # - the table of releases - are there any new ones?
     # - The table of persons/wallets - does it look like anything's been added there?
-    #   Top of bottom?
-    #
-    # Save As https://nbctf.mod.gov.il/en/Minister%20Sanctions/PropertyPerceptions/Pages/Blockchain1.aspx
-    # as 'Web Page, HTML only' in chrome to a temporary location e.g. /tmp/il_crypto.html.
-    # We use chrome because curl sometimes gets a bot blocking response.
-    #
-    # Then run `xmllint --format --html /tmp/il_crypto.html > datasets/il/mod_crypto/source.html`
-    # Then run `dos2unix datasets/il/mod_crypto/source.html`
-    # Then git diff --word-diff=color datasets/il/mod_crypto/source.html
-    # and see if there's anything in the content that's changed.
-    #
-    # Commit the changes so that we can see what changed from here next time.
+    # If updated, reflect changes in the Google Sheet and commit the new CSV.
+    write_csv_for_manual_diff(context, doc)
```

Comment:
it'd be nice to have this for both tables - I've seen updates in the top table, which were only added to the other table a week or so later.